news/2023/2023-12-22-how-advances-in-technology-affected-the-osu-community.md

```diff
@@ -0,0 +1,103 @@
+
+They quickly surged through the ranks after realizing that gaining pp by playing jump maps with a touch screen was simply easier given their point-to-point aim skill requirements. That however, did not just slip through without gathering negative attention from the community. The player base had turned against them, using them as a prime example of why touch screen was just unfair due to reasons stated before. However, it wouldn't be until November 2017 when freedomdiver decided to push touch screen to its limits, siding with the community and using actions instead of words to call for a change.
+
+The community somewhat sarcastically rallied around freedomdiver during one night in November, as they proceeded to break the pp record 3 times, all on the infamous [Yuima-ru*World TVver.](https://osu.ppy.sh/beatmapsets/459149#osu/994495) map. Hundreds of players spectated freedomdiver as they broke the pp record, and thanks to [CPUGeek](https://osu.ppy.sh/users/4172839) you can experience a snippet of the atmosphere surrounding the play:
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/JCU2ZWRj5gg" frameborder="0" allowfullscreen></iframe>
+</div>
+
+A month later, on December 15th, touch screen scores had their pp reduced along with an introduction of the brand new "Touch Device" mod, which would reduce the amount of pp gained from jumps on any map when osu! detected touch screen being used as a gameplay device.
+
+Nowadays, a new, unique usage for touch screen was figured. Some players use it exclusively as an aiming device, while still tapping to the beat with their good ol' keyboards. You can look at [bonk](https://osu.ppy.sh/users/4209965) and their [first person live play](https://www.youtube.com/watch?v=PE91EAzRcUo) to see what it looks like.
+
+## "Rapid trigger" feature on mechanical keyboards (2021)
+
+"Rapid trigger" is a new keyboard software feature that's first been implemented by Wooting, a mechanical keyboard manufacturer. Due to its analogue nature made possible by the [Hall effect](https://en.wikipedia.org/wiki/Hall_effect), it allows users to modify the position of any key's actuation point, or go beyond the need for a static actuation point altogether by using rapid trigger. The animation below visually represents how it works:
+
+![](/wiki/shared/news/2023-12-22-how-advances-in-technology-affected-the-osu-community/rapid_trigger_example.gif)
+
+This has caused quite the turmoil amongst the community, as some believe this greatly reduces the skill required to be considered "good at speed", and lowers the skill ceiling quite significantly by allowing players to be less precise with their technique, but still make inputs. The other side of the community argues that this is only beneficial for players who "mash" keys instead of trying to accurately hit notes.
```

Comment:
```suggestion
This has caused quite the turmoil amongst the community, as some believe this greatly reduces the skill required to be considered "good at speed", and lowers the skill ceiling quite significantly by allowing players to be less precise with their technique, but still not miss inputs or break combo as often. The other side of the community argues that this is only beneficial for players who "mash" keys instead of trying to accurately hit notes.
```